cirq/sim/clifford/clifford_simulator.py

```diff
@@ -336,94 +335,20 @@ def wave_function(self):
         return self.state_vector()
 
     def apply_unitary(self, op: 'cirq.Operation'):
-        if len(op.qubits) == 1:
-            self.apply_single_qubit_unitary(op)
-        elif isinstance(op, GlobalPhaseOperation):
-            self.ch_form.omega *= op.coefficient
-        elif op.gate == cirq.CNOT:
-            self.tableau._CNOT(self.qubit_map[op.qubits[0]],
-                               self.qubit_map[op.qubits[1]])
-            self.ch_form._CNOT(self.qubit_map[op.qubits[0]],
-                               self.qubit_map[op.qubits[1]])
-        elif op.gate == cirq.CZ:
-            self.tableau._CZ(self.qubit_map[op.qubits[0]],
-                             self.qubit_map[op.qubits[1]])
-            self.ch_form._CZ(self.qubit_map[op.qubits[0]],
-                             self.qubit_map[op.qubits[1]])
-        else:
+        from cirq.sim.clifford import (ActOnCliffordTableauArgs,
```

Comment:
why local import?

Reply:
Oh I guess the `*Args` are referring to this module?

Reply:
I had assumed so, but apparently I was wrong there